### ISSUE TITLE
🧱 ZUI: Text on picture, accessibility issue (design)

### ISSUE THREAD

**Issue by AVbiser** [zetkin/app.zetkin.org#2615]
Description: Low contrast with text on image (accessibility issue)
“Participate in the struggle for justice” is white text directly over a blurred image with varying contrast. This fails WCAG 2.1 contrast ratio guidelines.

<img width="1712" alt="Image" src="https://github.com/user-attachments/assets/a0091808-b0e4-49c5-bfaa-6bde61d88037" />

How to address this issue: Place a dark semi-transparent layer (overlay) between the background image and the white text.

Suggested solution: 

<img width="501" alt="Image" src="https://github.com/user-attachments/assets/0bee974a-e7f5-4094-8df8-062276b10345" />

**Comment by richardolsson** [comment]
Another good spot!

This page is part of the Gen2 (old) activist portal, which exists in another repo and has been deprecated.

The audit you're doing is really good and to get the most of it I suggest you focus on the Gen3 web app (the one in this repo) which you will recognize by the domain part of the URL. Domains starting with `app` (e.g. app.dev.zetkin.org or app.zetk.in) are running this web app.

**Comment by richardolsson** [comment]
This issue uses the "ZUI" tag and prefix incorrectly. I understand that it is very difficult to know what tags to use. I suggest avoiding tags and prefixes on the accessibility audit issues, and someone else can tag them in whatever way makes most sense to developers.
